src/js/format.js

```diff
@@ -75,6 +75,29 @@ const time = {
      */
     hasMeridian( time ) {
         return HAS_MERIDIAN.test( _cleanSpecialChars( time ) );
+    },
+    /**
+     * getTimezoneOffsetAsTime
+     */
+    getTimezoneOffsetAsTime() {
```

Comment:
This function has already been added to the Date prototype (not great practice but it's there): https://github.com/enketo/enketo-xpathjs/blob/master/src/date-extensions.js#L23, (so new Date().getTimezoneOffsetAsTime() I think) or this there a subtle difference?